### ISSUE TITLE
File Download missing spaces

### ISSUE THREAD

**Issue by chrisrhymes** [FutureNorthants/northants-design-system#334]
Testing tool picking up that there is no space between the last word of the actual download and the file format which is on the line beneath, so for instance on page https://www.westnorthants.gov.uk/schools-and-education/information-schools-staff/school-emergency-plans-and-procedures. 
Although it looks fine to anyone using the website, it may cause a problem to someone using a screenreader
